css-advance/electives.css

```diff
@@ -0,0 +1,108 @@
+.page-content {
+    font-family: 'Times New Roman', Times, serif;
+}
+
+.header {
+    height: 30px;
+    margin-right: 500px;
+    margin-bottom: 10px;
+}
+
+.header>button {
+    float: right;
+    margin-right: 8px;
+    margin-top: 8px;
+}
+
+.all-contents {
+    display: flex;
+    flex-direction: column;
+}
+
+.page-content {
+    display: flex;
+    flex-direction: row;
+}
+
+.elective-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+h2 {
+    color: #97bf0e;
+    margin: 0px
+}
+
+.info {
+    font-style: italic;
+}
+
+.button-color {
+    background-color: #97bf0e;
+    color: #ffffff;
+}
+
+.button-shape {
+    border: 1px solid #c3c3c2;
+    border-radius: 10px;
+    height: 30px;
+}
+
+.buttons-operation {
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+
+.navigation {
+    width: 300px;
+    float: left;
+    margin-right: 20px;
+    text-transform: uppercase;
+}
+
+.active {
+    background-color: #97bf0e;
+}
+
+.active>a {
+    color: #ffffff;
+}
+
+.sender-name {
+    color: #1e90ff;

```

Comment:
Blue color of text usually indicates url. You may want to change the mouse pointer to hand and style the text as underline on hover of the mouse over the name of the author. No need for a real url to be added (though it may make it easier to implement).